app/controllers/users_controller.rb

```diff
@@ -1,2 +1,26 @@
 class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      flash.now[:success] = t "success"
```

Comment:
```suggestion
      flash[:success] = t "success"
```
phân biệt flash và flash.now nào

Reply:
dạ
flash: chuyển hướng trang, rồi hiển thị
flash.now: chỉ hiển thị mẫu